aimm_adapters/scripts/file_handler.py

```diff
@@ -245,16 +256,37 @@ def iter_unique_keywords(path, tracked_set, start=False, count=False):
             continue
         if not filepath.is_file():
             # Explore subfolder for more labview files recursively
-            tracked_set, start = iter_unique_keywords(
-                filepath, tracked_set, start, count
+            tracked_set, start, collection = iter_unique_keywords(
+                filepath, tracked_set, start, count, collection
             )
             continue
         if filepath.suffix[1:].isnumeric():
             with open(filepath) as file:
                 column_names, column_size = parse_columns(file, no_device=True)
+
                 column_set = set(column_names)
                 if "Mono Energy" in column_set and column_size > 0:
                     if not count:
+                        if (
```

Comment:
It may be worth doing a set intersection here.

```py
excluded_columns = {"Ifluor", "IF", ...}
if not (excluded_columns & column_set):  # Check for any overlap